locales/en-US/editor.properties

```diff
@@ -4,8 +4,12 @@
 
 # General File I/O Error strings
 
+# Dialog Headers
+ERROR_DIALOG_HEADER=Error
```

Comment:
I couldn't find any string with the value "Error" or similar. The closest thing I found was "Genetic_error", but that string translate to "(error )" (which looks weird).